src/bulk/common.ts

```diff
@@ -868,6 +868,16 @@ export class FindOperators {
     this.bulkOperation.s.currentOp.arrayFilters = arrayFilters;
     return this;
   }
+
+  /** Specifies hint for UpdateOne or UpdateMany bulk operations. */
```

Comment:
Hint is used on deletes and replaces too (IIUC), maybe let's just be generic here and say it specifies the hint for the bulk operation

Reply:
Sure, done!